frontend/src/components/BudgetLineItems/CreateBLIsAndSCs/CreateBLIsAndSCs.hooks.js

```diff
@@ -409,6 +423,8 @@ const useCreateBLIsAndSCs = (
                     }
                     navigate(`/agreements/${selectedAgreement?.id}/budget-lines`);
                 } else {
+                    // TODO: Add logic to delete the agreement in the workflow
```

Comment:
Is this TODO required anymore? It looks like you did add the logic to delete the agreement in the lines below.